driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
I think to keep things really simple, I can just put a platform test in this test itself since I only need it for the 1 test.   This way we don't need to pollute `@CassandraVersion/@DseVersion/@CCMConfig` for the time being.  I'm sure we'll want it for more at some point, but by that time we'll be using the new test framework.